_utility-aliases.scss

```diff
@@ -1,6 +1,7 @@
 /* Border width
    ========================================================================== */
 
+$border-width--0:        $border-width--0;
```

Comment:
We mainly use words for the aliases. Is there a particular reason you're used `--0` here instead of `--zero` or `--none`?